.travis.yml

```diff
@@ -1,7 +1,8 @@
-sudo: false
+sudo: required
 language: node_js
 node_js:
 - "stable"
+- "0.12"
 - "0.10"
```

Comment:
It was there, I left it? Does anyone know of a way to lookup the current usage stats of the different Node versions? Might be interesting to see the current rate of download for the different versions from the Node.js website.